parsers/CA_SK.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import arrow
+import requests
+
+from .lib.exceptions import ParserException
+
+CONSUMPTION_URL = 'https://www.saskpower.com/ignitionapi/Content/GetNetLoad'
+SOURCE = 'saskpower.com'
+ZONE_KEY = 'CA-SK'
+
+
```

Comment:
Since data update frequency is [known to be ~ 5 minutes](https://github.com/electricityMap/electricitymap-contrib/pull/3587/files#r766983812) would adding the `refetch_frequency` decorator provide any value? Aside from its docstring description, how is the decorator used by Electricity Map's ingestion pipeline?

```suggestion

@refetch_frequency(datetime.timedelta(minutes=5))
```